test/e2e/virtual/workspaces/virtual_workspace_test.go

```diff
@@ -76,38 +82,52 @@ var testData = testDataType{
 }
 
 func TestWorkspacesVirtualWorkspaces(t *testing.T) {
```

Comment:
Do we need Parallel here too?

Reply:
doesn't harm, no?

But of course follow-up: use one fixture. 6 of them is little insane.

Reply:
fixed